### ISSUE TITLE
need to publish "materialstore-x.x.x.jar" to the MavenCentral repository from the materialstore as multi-project

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#267]
by #262, 263, 264, 265, I reorganized this project to be multi-module, 
- core
- base
- mapper
- tutorial

I would want to publish a new artifact

- `materialstore-core-x.x.x.jar`

to the MavenCentral repository.

On the other hand, I have this artifact already published in the MavenCentral.

- https://mvnrepository.com/artifact/com.kazurayam/materialstore

I want to retain this. And this should include the artifacts of `core`, `base`, `mapper` and `totorial` all-together.

**Comment by kazurayam** [comment]
the allJavadoc task is coded as:

```
task allJavadoc(type: Javadoc) {
    source exportedProjects.collect { project(it).sourceSets.main.allJava }
    classpath = files(exportedProjects.collect { project(it).sourceSets.main.compileClasspath })
    destinationDir = file("${buildDir}/docs/javadoc-all")
    failOnError(false)
}
```

Let me read the source and understand it.

The Javadoc task is described as:

- https://docs.gradle.org/current/dsl/org.gradle.api.tasks.javadoc.Javadoc.html

**Comment by kazurayam** [comment]
v0.9.1-SNAPSHOT now is able to publishToMavenLocal.

**Comment by kazurayam** [comment]
published [0.9.1](https://github.com/kazurayam/materialstore/tree/0.9.1)
